### PR TITLE
Listing Next.js as pre-requisite

### PR DESCRIPTION
**Pull request by alanjsebas**
As mentioned over Slack

## Summary by CodeRabbit

Documentation:
- Updated README.md to include a new prerequisite: NextJS must be installed on the system before contributing. This change ensures that potential contributors are better prepared and can contribute more effectively.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ There are two backends:
 
 ## Contribute
 
+Pre-requisites: Have NextJS installed on your system.
```
Comment (avikalpg) [inline README.md:27]:
```suggestion
Pre-requisites: 
- NextJS
```

Also, please add the command here to install Nextjs.

**Review by avikalpg — CHANGES_REQUESTED** [review]
Good PR, just a suggestion to make it a little better.

**Comment by alanjsebas** [comment]
I made the changes now

**Review by avikalpg — COMMENTED** [review]
If it can be installed by just `npm install`, why are we calling it a prerequisite? Then it is just part of the setup process, right?

**Comment by alanjsebas** [comment]
> If it can be installed by just `npm install`, why are we calling it a prerequisite? Then it is just part of the setup process, right?

You are right, should we just ignore it then? Or word it differently?

File: README.md [hunk 2]
```diff
@@ -24,6 +24,18 @@ There are two backends:
...
+
+-NextJS
+
+It can be installed through:
+
+```bash
+npm install
+# or
+npm install next
+```
```
Comment (avikalpg) [inline README.md:37]:
```suggestion
### Fork and clone repository 
@alanjsebas See how other projects right this.

### Install dependencies
Enter the repository folder and install all dependencies:

```bash
npm install
# or
yarn
```
```

**Comment by avikalpg** [comment]
So Alan, instead of saying that one needs to install NextJS, we can add a setup section in the "Contribute" section. I have added an outline for the same. We can ask the user to first fork the repository, then clone their fork of the repository, and then run `npm install` to install the dependencies (not just NextJS).

File: README.md [hunk 3]
```diff
@@ -24,17 +24,21 @@ There are two backends:
 
 ## Contribute
 
-Pre-requisites: 
+1. Fork or Clone Repository
```
Comment (avikalpg) [inline README.md:27]:
```suggestion
1. Fork and Clone Repository
```

**Review by avikalpg — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -24,6 +24,22 @@ There are two backends:
...
+   ```
+
+   Or
+
+   ```sh
+   yarn
+   ```
+
+3. Make changes, forked repositories can be used to submit pull requests to the original repository.
+
```
Comment (avikalpg) [inline README.md:42]:
```suggestion
### Fork and clone repository 
Fork this repository using the GitHub GUI.
Clone your fork of the repository:
```sh
git clone https://github.com/<your_username>/vibinex-server.git
```
or, if you use `SSH` cloning:
```sh
git clone git@github.com:<your_username>/vibinex-server.git
```

### Install dependencies
Enter the repository folder and install dependencies:
```sh
npm install
```

Or

```sh
yarn
```

**Review by avikalpg — APPROVED** [review]

**Pull request merged**